src/test/java/WebDriverScenarioWithTests.java

```diff
@@ -0,0 +1,122 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.logging.Logger;
+
+
+public class WebDriverScenarioWithTests {
+
+        private WebDriver driver;
+        Logger logger = Logger.getGlobal();
+
+        @BeforeMethod(alwaysRun = true)
+        public void browserSetup() {
+        driver = new ChromeDriver();
+    }
+
+        @Test(description = "JIRA binding can be here")
+        public void commonFlow() throws IOException {
+
+            ExpectedCondition<Boolean> pageLoadCondition = new
+                    ExpectedCondition<Boolean>() {
+                        public Boolean apply(WebDriver driver) {
+                            return ((JavascriptExecutor) driver).executeScript("return document.readyState").equals("complete");
+                        }
+                    };
+            WebDriverWait wait = new WebDriverWait(driver, 10);
+
+        driver.get("http://google.com");
+        driver.manage().window().maximize();
```

Comment:
after this step wait need to be added, because page can not to fully load and next element can not be found.

Reply:
Thank you, added.
Please see in the following commit.